src/styles/components/sidebar.scss

```diff
@@ -195,3 +205,97 @@ $sidebar-item-font-size: 14px;
 		}
 	}
 }
+
+.modal {
+	.modal-dialog.left, {
+		position: fixed;
+		margin: auto;
+		width: 200px;
+		height: 100%;
+		-webkit-transform: translate3d(0%, 0, 0);
+		-ms-transform: translate3d(0%, 0, 0);
+		-o-transform: translate3d(0%, 0, 0);
+		transform: translate3d(0%, 0, 0);
+	}
+
+	.modal-dialog.left {
+		.modal-content {
+			height: 100%;
+		}
+	}
+
+	.modal-content {
+		display: block;
+		overflow: hidden;
+	}
+
+	.modal-dialog.left {
+		.modal-body {
+			padding: 15px 15px 80px;
+		}
+	}
+
+	/*Left*/
+	&.fade {
+		.modal-dialog.left {
+			left: -200px;
+			-webkit-transition: opacity 0.3s linear, left 0.3s ease-out;
+			-moz-transition: opacity 0.3s linear, left 0.3s ease-out;
+			-o-transition: opacity 0.3s linear, left 0.3s ease-out;
+			transition: opacity 0.3s linear, left 0.3s ease-out;
+		}
+	}
+	&.fade.show {
+		.modal-dialog.left {
+			left: 0;
+		}
+	}
+}
+
+@media screen and (max-width: 800px) {
+	.mobile-sidebar-burger {
+		color: $toggle-text-color;
+		visibility: visible;
+		font-size: 1.5rem;
+		margin-left: auto !important;
+		cursor: pointer;
+		position: relative;
+
+		& [class^="cil-"] {
+			position: absolute;
+			top: 20px;
+			left: 32px;
```

Comment:
@Aringoaway To me it looks like the hamburger menu has bigger left padding than the `chevron` icon's (opening the collapsed header items) right padding. There is a disproportion.

Reply:
@Pe5h4 fixed it